packages/rn-tester/README.md

```diff
@@ -8,9 +8,16 @@ Before running the app, make sure you ran:
 ```sh
 git clone https://github.com/facebook/react-native.git
 cd react-native
+```
+
+It's strongly suggested to use [nvm](https://nvm.sh) (Node Version Manager), rather than install `yarn` into your operating system's global `node_modules`.
+
+```
+npm install -g yarn
```

Comment:
This is not actually true.. Where did you find this information? Personally, I'm not using any of them.
Although it is considered a good practice to use version managers, in general, we don't want to add too many requirements for our users, so we'd rather drop this.